backend/codesirius/problems/models/submission.py

```
@@ -0,0 +1,26 @@
+    """
+
+    user = models.ForeignKey(
+        get_user_model(),
+        on_delete=models.CASCADE,
+    )
+
+    class Meta:
+        verbose_name = "Submission"
+        verbose_name_plural = "Submissions"
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)